#!/usr/bin/env python
# field of view configuration

PACKAGE="cob_3d_mapping_geometry_map"
import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("file_path", str_t, 1, "File path to store data", "")
gen.add("save_to_file", bool_t, 2, "Should data be stored", False)
gen.add("cos_angle", double_t, 3, "angle threshold for map merging", 0.97, 0.7, 1);
gen.add("d", double_t, 3, "distance threshold for map merging", 0.1, 0, 0.5);
gen.add("map_frame_id", str_t, 4, "frame id for tf", "/map")
gen.add("enable_tf" , bool_t, 2, "enable transform to target frame",True)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "geometry_map_node"))
